src/web/src/assets_in.ecpp

```diff
@@ -137,6 +138,8 @@ bool database_ready;
     std::string without = "";
     std::string status = "";
     std::string details = "";
+    std::string configured = "";
+    std::vector<std::string> checked_configured = { "all", "no", "yes" };
```

Comment:
empty value is not allowed?  or 'all' eq. empty?  (the common-db api don't handled 'all' value for the configured arg)

Reply:
yes the "all" value (like empty) deactivate the option